recipe/build-cxx.sh

```diff
@@ -38,19 +32,14 @@ ${BUILD_PREFIX}/bin/cmake ${CMAKE_ARGS} ${ARCH_ARGS} \
   -D CMAKE_CXX_COMPILER=${CXX} \
   -D CMAKE_CXX_FLAGS="${CXXFLAGS}" \
   -D CMAKE_INSTALL_LIBDIR=lib \
-  -D EINSUMS_STATIC_BUILD=OFF \
+  -D EINSUMS_WITH_TESTS=OFF \
```

Comment:
do you want to run tests on the build to ensure it's working ok?

Reply:
Whoops. I'll add that back. :)